docs/products/cassandra/get-started.md

```diff
@@ -1,44 +1,36 @@
 ---
 title: Get started with Aiven for Apache Cassandra®
+sidebar_labels: Get started
 ---
 
-The first step in using Aiven for Apache Cassandra® is to create a
-service. You can do so either using [Aiven
-Console](https://console.aiven.io/) or the [Aiven
-CLI](https://github.com/aiven/aiven-client).
+Quick and easy way to start using Aiven for Apache Cassandra®: create a service either in the [Aiven Console](https://console.aiven.io/) or with the Aiven CLI.
```

Comment:
That's not for us to decide :) Let's never judge complexity please.  (applies to all files of the PR)

```suggestion
Start using Aiven for Apache Cassandra® by creating a service either in the [Aiven Console](https://console.aiven.io/) or with the Aiven CLI.
```